pkgs/development/interpreters/babashka/wrapped.nix

```diff
@@ -16,26 +18,42 @@
   # https://github.com/NixOS/nixpkgs/pull/248207
 , withRlwrap ? false
 }:
-stdenvNoCC.mkDerivation (finalAttrs: {
+stdenvNoCC.mkDerivation (finalAttrs: rec {
   pname = "babashka";
-  inherit (babashka-unwrapped) version meta doInstallCheck installCheckPhase;
+  inherit (babashka-unwrapped) version meta doInstallCheck;
 
   dontUnpack = true;
   dontBuild = true;
 
   nativeBuildInputs = [ makeWrapper ];
 
+  # Can't use the release source tarball for this due to
+  # https://github.com/babashka/babashka/issues/1627 (i.e. the `deps.clj` submodule is not included)
+  src = fetchFromGitHub {
+    owner = "babashka";
+    repo = "babashka";
+    rev = "v${version}";
+    hash = "sha256-xwc+IjaMUPPWxFf0oKOkB+ixgZ9sXfci1rioX050e6o=";
+    fetchSubmodules = true;
+  };
+
+  clojureToolsVersion = lib.removeSuffix "\n" (builtins.readFile "${src}/deps.clj/resources/DEPS_CLJ_RELEASED_VERSION");
```

Comment:
That is probably triggering IFD which would make ofborg fail

Reply:
Interesting! What in particular about this expression is causing this and where could I read up on IFD?

Reply:
You are trying to read a file into nix code that is from a fixed output derivation.

Maybe https://nixos.wiki/wiki/Import_From_Derivation ?